src/fetchers.ts

```diff
@@ -0,0 +1,88 @@
+import Appstream from './types/Appstream'
+import Collections, {Collection} from './types/Collection'
+import Category from './types/Category'
+
+import {POPULAR_URL, APP_DETAILS, RECENTLY_UPDATED_URL, EDITORS_PICKS_APPS_URL, EDITORS_PICKS_GAMES_URL, APPSTREAM_URL, CATEGORY_URL, SEARCH_APP} from './env'
+
+const fetchEntry = async (entry: string): Promise<Appstream | {}> => {
+  let entryJson: Appstream | {};
```

Comment:
There's a weird combination of either using `;` or not, we should setup a linter and keep the code consistent on that regard

Reply:
Yeah, unfortunately I have a habit of putting semicolon by myself, but some linter and maybe prettier is a cool idea, will create issue for that.

Reply:
Aha, I tend to use the standardjs linter which doesn't use the semicolon :-) but I don't mind that much, it's just details. We  can set that at another time.